README.md

```diff
@@ -108,7 +108,7 @@ Some useful flags are:
   --runs: The number of benchmark runs.
     (default: '3')
     (an integer)
-  --upload_data_to: The details of the BigQuery table to upload results to: <project_id>:<dataset_id>:<table_id>:<location>
+  --upload_to_bigquery: The details of the BigQuery table to upload results to: <project_id>:<dataset_id>:<table_id>:<location>
```

Comment:
mention the new flag here (and below) as well?

Reply:
Done